docs/antora-playbook.yml

```diff
@@ -0,0 +1,29 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/asciidoctor/asciidoctor-intellij-plugin/main/src/main/resources/jsonSchemas/antoraPlaybookSchema.json
+
+site:
+  title: Reqstool Demo Documentation
```

Comment:
Just Reqstool Demo? (it's not a Demo of Documentation)

Reply:
Ok